test/browser/svg.test.js

```diff
@@ -29,6 +29,22 @@ describe('svg', () => {
 		`.replace(/[\n\t]+/g,'')));
 	});
 
+	it('should support svg attributes', () => {
```

Comment:
This test passes even without using any of the `*NS` functions. Maybe we should add a test with an attribute that has a namespace. Something like `xlink:href` which I think was the why @developit found out that we lack the `*NS` functions.

_Note: mdn lists [xlink:href](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href) as being deprecated in SVG2 in favour of just `href`, but trouble is that no browser seem to really support SVG2 at this point._